packages/contracts/contracts/tcr/CivilTCR.sol

```diff
@@ -56,21 +58,23 @@ contract CivilTCR is RestrictedAddressRegistry {
   mapping(uint => Appeal) public appeals; // map challengeID to appeal
 
   /**
-  @notice Contructor Sets the addresses for token, voting, parameterizer, appellate, and fee recipient
+  @notice Init function calls AddressRegistry init then sets IGovernment
   @dev passes tokenAddr, plcrAddr, paramsAddr up to RestrictedAddressRegistry constructor
   @param tokenAddr Address of the TCR's intrinsic ERC20 token
   @param plcrAddr Address of a PLCR voting contract for the provided token
   @param paramsAddr Address of a Parameterizer contract
   @param govt IGovernment contract
   */
-  function CivilTCR(
+  function initTCR(
```

Comment:
Why is this not a constructor anymore? All other functions in this smart-contract would have to verify if it it was initialised beforehand to be able to work with known state.

Please make it into a constructor again

Reply:
this brings it in line with the standard implementation. asking consensys TCR to explain reasoning behind init vs constructor, will get back to you